icu4j/perf-tests/collationperf.pl

```diff
@@ -42,10 +42,10 @@
 
 my $OS=$^O;
 my $CLASSPATH;
-if ($OS eq "linux" || $OS eq "darwin") {
-	$CLASSPATH = './target/*:./target/dependency/*';
-} else {
+if ($^O eq "MSWin32") {
```

Comment:
optional: use the alias `$OS` that you defined above (presumably to be more human readable) rather than `$^O`